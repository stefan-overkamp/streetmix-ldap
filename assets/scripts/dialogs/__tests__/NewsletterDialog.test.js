/* eslint-env jest */
import React from 'react'
import { renderWithReduxAndIntl } from '../../../../test/helpers/render'
import NewsletterDialog from '../NewsletterDialog'

// Mock Mailchimp HTML snippet
jest.mock('../Newsletter/mailchimp.html', () => '<div>foo</div>')

describe('NewsletterDialog', () => {
  it('renders snapshot', () => {
    const { asFragment } = renderWithReduxAndIntl(<NewsletterDialog />)
    expect(asFragment()).toMatchSnapshot()
  })
})
