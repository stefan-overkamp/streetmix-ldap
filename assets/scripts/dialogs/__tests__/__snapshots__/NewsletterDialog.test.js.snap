// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsletterDialog renders snapshot 1`] = `
<DocumentFragment>
  <div
    class="dialog-box-container"
  >
    <div
      class="dialog-box-backdrop"
    />
    <div
      class="dialog-box"
      role="dialog"
    >
      <button
        class="close"
        title="Dismiss"
      >
        <svg
          class="octicon"
        />
      </button>
      <div
        class="newsletter-dialog"
      >
        <header>
          <h1>
            Subscribe to our newsletter
          </h1>
        </header>
        <div
          class="dialog-content"
        >
          <div>
            <div>
              foo
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
